### DISCUSSION TITLE
How to enter your credentials?

### DISCUSSION THREAD

**Discussion by mrjeth** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#449] [Q&A]
First of all thank you for a very interesting project!
I am a total new to Platformio and Github so please forgive me if it is obvious to you
My problem:
I have tried to followed the build instructions and build, uploaded the code and the file system. According to the instructions.

However, in 8)  We can read:
“After upload. (Did you upload the data files as well?!) 
An Access Point is created called "Lay-Z-Spa Module". Pswd "layzspam0dule".
Log in and enter your wifi credentials. 
Visit http://layzspa.local and click on the hamburger menu in the top right 
corner.”

What does it mean: How should I connect to http://layzspa.local?
It clearly doesn’t work via a web browser !
Please guide me into this

In the Terminal windows I get:

"Failed to read cmdq.txt
Failed to read wifi.json. Using defaults.
WiFi > Using WiFiManager Config Portal
*WM: 
*WM: AutoConnect
*WM: Connecting as wifi client...
*WM: Status:
*WM: 0
*WM: No saved credentials
*WM: Connection result:
*WM: 0
*WM:
*WM: Configuring access point...
*WM: Lay-Z-Spa Module
*WM: layzspam0dule
*WM: AP IP address: 
*WM: 192.168.4.1
*WM: HTTP server started"

**Comment by visualapproach** [comment]
> An Access Point is created called "Lay-Z-Spa Module".

Look for an access point with SSID "layzspa". (Search for new Wi-Fi stations)

**Comment by mrjeth** [comment]
Hi Thank you very much! Thomas? Of course how stupid of me! I have been sitting here for several hours without any progress!  Where did you get the inspiration from setting up the Project? The Nodemcu Wifi examples are normally very basic and is not so advanced as your code. /Tomas

**Reply by visualapproach** [reply]
Ingen orsak Tomas! Well, it's been growing over time. First I just wanted to hack the interface to be able to control it. Then I needed to learn how to do communication to and from a web page. And you're right, most tutorials are pretty basic, but trial and error and google solved most of it. There is a bunch of things involved, like basic electronics, digital communication (SPI, serial etc), html, c++ and more, of which I had poor to basic knowledge of but I've learned a lot which feels great. There has ofc been requests and contributions from other people too!
